internal/manifests/internal/gateway/options.go

```diff
@@ -11,4 +11,14 @@ type Options struct {
 	Namespace        string
 	Name             string
 	StorageDirectory string
+
+	GatewaySecret []*Secret
```

Comment:
This is a slice of secrets, thus deserves plural case, i.e. `GatewaySecrets` or `TenantSecrets`. I am in favor of later as the whole Struct is already referenced as `gateway.Options`